spec/command/init_spec.rb

```diff
@@ -49,15 +49,15 @@
 
     context "auto-fetching fingerprint" do
       before {
-        allow_any_instance_of(HighLine).to receive(:ask).with("Enter the hostname (and optional port) of your Conjur endpoint: ").and_return "the-host"
+        allow_any_instance_of(HighLine).to receive(:ask).with("Enter the URL of your Conjur service: ").and_return "http://host.example.com"
         allow(Conjur::Command::Init).to receive_messages get_certificate: ["the-fingerprint", nil]
         allow_any_instance_of(HighLine).to receive(:ask).with(/^Trust this certificate/).and_return "yes"
       }
 
       describe_command 'init' do
-        it "fetches account and writes config file" do
-          # Stub hostname
-          expect(Conjur::Core::API).to receive(:info).and_return "account" => "the-account"
+        it "writes config file" do
+          expect_any_instance_of(HighLine).to receive(:ask).with("Enter the URL of your Conjur service: ").and_return "http://host.example.com"
```

Comment:
It's the way the other specs were written here. I think it's ok, for now at least.